us-31 Frontend Readme (#1)
solid-stack-solutions/voycar-frontend: README.md

# Overview

> //Todo: (semi-)short Desciption of the frontend

Reviewer: ```suggestion
//Todo: (semi-)short description of the frontend
```

> //Todo: (semi-)short Desciption of the frontend

Reviewer: ```suggestion
//Todo: (semi-)short description of the frontend
```

# How to use

> //Todo: short description on how to use the frontend Pages, maybe screenshot/video showcase

Reviewer: ```suggestion
//Todo: short description on how to use the frontend pages, maybe screenshot/video showcase
```

# Installation

> //Todo: explenation of the installation of the fronend 

Reviewer: ```suggestion
//Todo: explanation of the installation of the frontend
```

## Dependencies

> //Todo: list big dependencies (used Framworks etc.) with short description + how to install them with the right setup

Reviewer: ```suggestion
//Todo: list big dependencies (used Frameworks etc.) with short description + how to install them with the right setup
```

# Functionality
//Todo: what the frontend does, what it's used for
# Deployment
//Todo: docker stuff 
# Development
//Todo: introduction on how the development of any new, fixed or refactored parts of the frontend

> //Todo: Link to documentation of the frontend

Reviewer: ```suggestion
//Todo: link to documentation of the frontend
```

## Styling
//Todo: list styling conventions
### Page Styling

> //Todo: List of styling conventions for the Pages (colorpalette, layouts, common icons, etc.)

Reviewer: ```suggestion
//Todo: list of styling conventions for the pages (colorpalette, layouts, common icons, etc.)
```

### Code Styling

> //Todo: List of styling conventions

Reviewer: ```suggestion
//Todo: list of styling conventions
```

## Conventions
//Todo list general conventions for development

> //Todo: helpfull informations and how-to's for Development

Reviewer: ```suggestion
//Todo: helpful information and how-to's for development
```
